src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx

```diff
@@ -285,21 +305,26 @@ const EnableWhenCondition: React.FC<EnableWhenConditionProps> = ({
           </Stack>
         </Grid>
       </Grid>
-      <Box sx={{ mt: 2 }}>
-        <Typography typography='body2' fontWeight={600}>
-          Advanced Options
-        </Typography>
-        <Stack>
-          <LabeledCheckbox
-            label='Compare with a Local Constant instead of a Dependent Question'
-            checked={advanced.localConstant}
-            sx={{ width: 'fit-content' }}
-            onChange={(evt, checked) =>
-              setAdvanced((old) => ({ ...old, localConstant: checked }))
-            }
-          />
-        </Stack>
-      </Box>
+      {/* TODO: Add typing for local constants (treat `today` as a date)
+      and add other useful local constants such as project ID.
+      Hiding this behind a super-admin-only curtain for now.*/}
+      <RootPermissionsFilter permissions='canAdministrateConfig'>
```

Comment:
Since we don't plan to implement typing for local constants for V1 (https://github.com/open-path/Green-River/issues/6403), and the only 2 local constants available are today's date and age 18 dob, I'm proposing to hide it behind super-admin-only permissions for now.

Another possible v1 stopgap would be to provide instructions for how to format the date correctly in the helper text, but I think this is simpler.